xtend-contrib/src/main/java/de/oehme/xtend/contrib/Builder.xtend

```diff
@@ -54,6 +67,15 @@ class BuilderProcessor extends AbstractClassProcessor {
 					'''
 					primarySourceElement = cls
 				]
+				addMethod('''set«builtField.simpleName.toFirstUpper»''') [
```

Comment:
@szarnekow Aha, it's a bug, but here we get an error because the Builder class has a field with that name and it is private.